app/models/concerns/hyrax/ability.rb

```diff
@@ -188,5 +193,15 @@ def user_is_depositor?(document_id)
       def curation_concerns_models
         [::FileSet, ::Collection] + Hyrax.config.curation_concerns
       end
+
+      def can_review_submissions?
+        # Are there any workflows where this user has the "approving" responsibility
+        approving_role = Sipity::Role.find_by(name: 'approving'.freeze)
+        return false unless approving_role
+        Hyrax::Workflow::PermissionQuery.scope_processing_agents_for(user: current_user).any? do |agent|
```

Comment:
I'm wondering why this is hidden for other people? Instead of asking do I have the "approving" role, which may not exist on some workflows, why not always render this page?

Anyone may have one or more roles for any of the submissions - they may have the approving role, or the metadata approving role, or copyright reviewing role. And shouldn't they, in all cases see the page for reviewing submissions?